docs/workspaces/index.md

```diff
@@ -221,6 +221,16 @@ If a file name appears in multiple locations, the following precedence takes eff
 
 - Files in the `workspace/` directory are considered last.
 
+## Maintained workspace images
+
+PrairieLearn provides and maintains the following workspace images:
+
+- [prairielearn/workspace-desktop](https://github.com/PrairieLearn/PrairieLearn/tree/master/workspaces/desktop/): An Ubuntu 24.04 desktop
```

Comment:
Please use backticks for image names like in the example I gave:

```suggestion
- [`prairielearn/workspace-desktop`](https://github.com/PrairieLearn/PrairieLearn/tree/master/workspaces/desktop/): An Ubuntu 24.04 desktop
```

Ditto for images below and in other files.

Reply:
Ah yes, sorry about that